tests/generics/toverloading_typedesc.nim

```diff
@@ -1,6 +1,6 @@
 discard """
   exitcode: 0
-  disabled: '''true'''
+    disabled: '''true'''
```

Comment:
Crreated around 2015-06 in a commit that references #3022
Disabled around 2017-11 for CI sake?
Working in my pc.

Reply:
Enable